### PR TITLE
show home page

### PR DESCRIPTION
**Pull request by caovanson06112001**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52399)

## WHAT (optional)
- [show home page]

## HOW
- Add css, js for home page ,show data book

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
Screenshot from 2022-08-16 10-11-58

### CODE REVIEW

**Comment by caovanson06112001** [comment]
ready

File: app/views/shares/website/_header.html.erb [hunk 1]
```diff
@@ -67,50 +67,50 @@
...
-                <% end %>
-              </li>
-              <li>
-                <%= link_to options = {locale: :en}, class:"dropdown-item" do %>
-                  <i class="feather icon-lock"></i> <%= t "english" %>
+            </a>
+            <div class="dropdown-menu dropdown-menu-end profile-notification">
+              <div class="pro-head">
+                <%= image_tag "user/avatar.png", alt: "user", class: "img-radius wid-40" %>
+                <span>John Doe</span>
```
Comment (ngocvt-0484) [inline app/views/shares/website/_header.html.erb:79]: để tên user đang đăng nhập đi e

**Comment by caovanson06112001** [comment]
ready

**Comment by manhnt-0960** [comment]
conflicted

**Comment by caovanson06112001** [comment]
ready

**Pull request merged**